eng/.docsettings.yml

```diff
@@ -1,160 +1,162 @@
 omitted_paths:
```

Comment:
Unclear why this file got re-indented, but can you restore it and just commit the added line?